sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,237 @@
+- Does the proposed sBTC signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer? 
+  * Note: This metric may be self-affirmed if independent verification is not possible, or confirmed by on-chain voting/stacking activity.*
+- Does the proposed sBTC signer commit to a direct communication channel to be set up with the sBTC core engineers in order to respond to urgent updates within 24 hours?
+- Has the signer made contributions to Bitcoin or the Stacks network over the past year that demonstrate their commitment to the growth and success of the network? Examples include, but are not limited to: publishing independent research, marketing, co-authoring a SIP, submitting a Stacks pull request/issue, providing feedback on Stacks core development, or contributing to a Stacks Working Group.
+
+The criteria described above will be used to identify sBTC Signers that are able to meet some or all of the responsibilities described in the previous section.
+
+### Selection Process
+The initial sBTC Signer Set will be finalized from the list of eligible Signers, based on the above criteria. The results will be published as a discussion in the [sBTC Working Group](https://github.com/stacks-network/sbtc/discussions/624).
+
+The selection process is as follows:
+1. **Nomination Phase**: Open a call for nominations within the community.
+2. **Evaluation & Community Feedback**: The proposed signer set will be published to provide transparency.
+3. **SIP Vote**: The community will vote on the signer criteria and proposed signer set.
+4. **Final Selection**: The final list of signers will be selected based on community feedback and successful completion of SIP-028.
+
+### Updating The sBTC Signer Set
+In the event that the sBTC Signer Set needs to be updated (for example, if a signer is no longer available to complete their responsibilities) sBTC Signers can perform a threshold vote to agree on the updated set. This process will also be performed if a signer needs to rotate their cryptographic keys.
+
+
+## Related Work
+
+### [WBTC](https://wbtc.network/assets/wrapped-tokens-whitepaper.pdf)
+
+WBTC is a closed membership system. It is made up of 50+ merchants and custodians with keys to the WBTC multisig contract on Ethereum. WBTC deposits and withdrawals can only be performed by the authorized merchants, and end users purchase WBTC directly from the merchants. Although the merchants manage issuance and redemption, all BTC backing WBTC is held by a single custodian.
+
+### [tBTC v2](https://whitepaper.io/document/691/tbtc-whitepaper)
+
+tBTC is an open membership system, where the BTC is managed by a rotating set of randomly selected nodes which manage a threshold wallet. The system requires that 51-of-100 randomly selected wallet signers must collaborate to produce a proper signature.
```

Comment:
Can you explain more about how sBTC differs from tBTC?  In what way(s) is sBTC "better," and how do we know so?

Reply:
The main difference is in the finality assumptions. tBTC relies on Ethereum's network security, which doesn't recognize Bitcoin. On the other hand, sBTC forks with Bitcoin and inherits finality from Bitcoin. This allows sBTC to provide faster deposit/withdrawal times compared to tBTC (this is described in the SIP).